dcos/dcos-ui
# DCOS-54491: Framework configuration form tab bug on Firefox <=63 on Windows/Linux

## Testing
1. Open Firefox version 63 or older on a Windows or Linux machine (if you need an invite to our BrowserStack account, ping me)
2. Go to the Catalog, pick a framework, and click "Review & Run" to open the framework configuration form

The tabs on the left should all be clickable

## Trade-offs
I wasn't able to find what exactly changed between 1.12 and 1.13 that surfaced this bug, so this is just a quick fix

## Dependencies



## Screenshots
Before:
![FrameworkConfigFFTabFix_before](https://user-images.githubusercontent.com/2313998/58600349-51a1d400-8252-11e9-8b29-7ae94ffdca05.gif)


After:
![FrameworkConfigFFTabFix_after](https://user-images.githubusercontent.com/2313998/58600368-5d8d9600-8252-11e9-8160-c60dec5c2e97.gif)
